### ISSUE TITLE
Adding websites to the allow list does not work.

### ISSUE THREAD

**Issue by tc0566** [AdguardTeam/AdguardBrowserExtension#2221]
After adding a website to the allow list ads are still being blocked. This issue was listed here because adguard bot listed to an open issue with a screenshot and was closed with no fix.

Information 

AdGuard product: | AdGuard Browser extension v4.1.39
Browser: | Chrome
Stealth mode: | disabled
Filters: | Ad Blocking:AdGuard BaseOther:Filter unblocking search ads and self-promotion

![facebook adblock](https://user-images.githubusercontent.com/83817724/198755162-798897d7-c0f4-4c88-ad25-55eed2441d7d.jpg)

https://github.com/AdguardTeam/AdguardFilters/issues/133386

**Comment by maximtop** [comment]
Hi, @tc0566. Could you specify please what site are you adding to the allowlist? 
Could you send your settings?

**Comment by tc0566** [comment]
The pic as you can see is blocking game ads and still blocking ads on Facebook. It simply does work adding a website to the list. I have to disable AdGuard every time I open the game.. Beefore the last update I was able to play the video ads on the game and only had the game itself on the list and not entire Facebook added to the list. The last screenshot was just done with AdGuard on. While the game was loading it blocked 4 elements and as you can see it will not let me play the video.     On Monday, October 31, 2022, 06:48:13 AM EDT, Maxim Topciu ***@***.***> wrote:  
 
 


Hi, @tc0566. Could you specify please what site are you adding to the allowlist?
Could you send your settings?

**Comment by maximtop** [comment]
I didn't get your settings. Which version did you have before?

Unfortunately, I can't test on my side, because this game is not working on my side with and without an extension 
<img width="703" alt="image" src="https://user-images.githubusercontent.com/17272769/198995905-2203ca73-ff7c-4bc6-b5c6-1b9212841c78.png">

**Comment by tc0566** [comment]
The only thing different changed in any settings is Facebook and the game added to the allow list. Doesn't really matter any site added to the allow list still blocks all ads. As you can see from latest pic done just now it has blocked 82 elements on Facebook with Facebook on the allow list.     On Monday, October 31, 2022, 07:19:57 AM EDT, Maxim Topciu ***@***.***> wrote:  
 
 


I didn't get your settings. Which version did you have before?

Unfortunately, I can't test on my side, because this game is not working on my side with and without an extension

**Comment by tc0566** [comment]
Here is the filtering log set to all and you can clearly see it is blocking elements on Facebook.     On Monday, October 31, 2022, 07:19:57 AM EDT, Maxim Topciu ***@***.***> wrote:  
 
 


I didn't get your settings. Which version did you have before?

Unfortunately, I can't test on my side, because this game is not working on my side with and without an extension

**Comment by tc0566** [comment]
AdGuard Browser extension
Version 4.1.39
     On Monday, October 31, 2022, 07:19:57 AM EDT, Maxim Topciu ***@***.***> wrote:  
 
 


I didn't get your settings. Which version did you have before?

Unfortunately, I can't test on my side, because this game is not working on my side with and without an extension

**Comment by maximtop** [comment]
> Doesn't really matter any site added to the allow list still blocks all ads.
Maybe you have inverted allowlist?

> Here is the filtering log set to all and you can clearly see it is blocking elements on Facebook
Seems like you try to send images via email client, but they do not get to the github issue https://github.com/AdguardTeam/AdguardBrowserExtension/issues/2221

> Version 4.1.39
This is the last version, but you've said that version before.

**Comment by tc0566** [comment]
Just forget it.. I'll find another adblocker with a whitelist that actually works.  You can clearly see the issue in the pics you were sent via email. And all you can suggest is that is that maybe I have the filter inverted. No it's not inverted. The allow list does not work. That is the issue.
![facebook adblock](https://user-images.githubusercontent.com/83817724/199063428-df830461-bcd0-4408-afa2-7a213a5c4bcd.jpg)
![Fb Ad](https://user-images.githubusercontent.com/83817724/199063501-9dae3db1-e4bf-44b3-a627-8ce3d161a50a.jpg)
![FB block log](https://user-images.githubusercontent.com/83817724/199063533-bdd5bd2d-ee87-4a57-8b57-686d91e472ab.jpg)
![Fb game](https://user-images.githubusercontent.com/83817724/199063559-22af2b54-4e92-4c15-b483-da70a8bd2ff6.jpg)
![settings](https://user-images.githubusercontent.com/83817724/199063580-89a3b754-de2d-4ab7-a439-ae90c46525a7.jpg)

**Comment by maximtop** [comment]
There is no sense to use urls in the allowlist, it is only for domain names
Try to remove it, save and check if page works now.

**Comment by tc0566** [comment]
No it still does not work. The only thing that works is turning off AdGuard. Now I'm removing it because clearly your allow list no longer works. It was working before your last update on October 24th. The fact that you continue to ignore that fact tells me I'm clearly having a conversation with the wrong person.
     On Monday, October 31, 2022, 01:14:58 PM EDT, Maxim Topciu ***@***.***> wrote:  
 
 


There is no sense to use urls in the allowlist, it is only for domain names
Try to remove it, save and check if page works now.

**Comment by tc0566** [comment]
I love the fact that your showing me this on an older version that does work. I can clearly see the version number in yours is 4.1.34.  This is a working version. The current version that no longer works is 4.1.39 How about you try doing this on the current version that no longer works. 
The only thing I see in the log is shown in the pic
![console 2](https://user-images.githubusercontent.com/83817724/199074918-5b8b8f34-0466-4c13-9737-2264be7ed12d.jpg)

**Comment by tc0566** [comment]
The filtering engine is not ready... probably because it's broken in the latest version. Maybe you should be testing the latest version. The one you are using is older than mine and is not listed in the changelog.

**Comment by maximtop** [comment]
> The filtering engine is not ready

it is not an error, it is just information on the extension start

I've tested on the latest extension as well
<img width="1250" alt="image" src="https://user-images.githubusercontent.com/17272769/199209054-fc0f428d-933a-47f6-84d5-660aa46ad79d.png">

<img width="1447" alt="image" src="https://user-images.githubusercontent.com/17272769/199209126-46039836-4915-4e80-9c8d-8f3a1d663c5d.png">

<img width="1099" alt="image" src="https://user-images.githubusercontent.com/17272769/199209210-59f79798-4c56-4f6f-828a-6fda91261ed7.png">

Try to install the extension in another browser and test if the allowlist works there.

**Comment by tc0566** [comment]
I sent you what it shows on, mine every site added to the allow list, still blocks everything. Checked it with multiple sites. It does not save to the allow list now as soon as you go to a site on the allow list it removes the entry. Also now the log showing the sites allowed or not allowed is blank for every page. i am done with this ad blocker, it simply no longer works.     On Tuesday, November 1, 2022, 06:05:07 AM EDT, Maxim Topciu ***@***.***> wrote:  
 
 



The filtering engine is not ready


it is not an error, it is just information on the extension start

I've tested on the latest extension as well






Try to install the extension in another browser and test if the allowlist works there.
